### PR TITLE
Return regionId in getIdentity calls

### PR DESCRIPTION
**Pull request by jonsnyder**
## Description

Konductor returns edge information in the header "x-adobe-edge". This change parses this header and exposes the regionId as part of the response to the getIdentity command.

## Related Issue

https://jira.corp.adobe.com/browse/PDCL-4243

## Motivation and Context

Some customers need the regionId to call other services.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Improvement (non-breaking change which does not add functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [x] I have made any necessary test changes and all tests pass.
- [x] I have run the Sandbox successfully.

### CODE REVIEW

**Review by jfkhoury — APPROVED** [review]
💡

File: src/components/Identity/createComponent.js [hunk 1]
```diff
@@ -28,6 +30,7 @@ export default ({
             setLegacyEcid(ecid);
           }
         }
+        edge = assign(edge, response.getEdge());
```
Comment (Aaronius) [inline src/components/Identity/createComponent.js:33]: Mind leaving a comment as to why we're using `assign` here?

**Review by Aaronius — CHANGES_REQUESTED** [review]
Love it. I'd like for the change I requested to be made, but let me know if you disagree so we can talk through it.

File: src/core/edgeNetwork/injectSendEdgeNetworkRequest.js [hunk 2]
```diff
@@ -75,10 +75,10 @@ export default ({
           .call({ error })
           .then(throwError, throwError);
       })
-      .then(({ parsedBody, edge }) => {
+      .then(({ parsedBody, getHeader }) => {
```
Comment (jfkhoury) [inline src/core/edgeNetwork/injectSendEdgeNetworkRequest.js:78]: Love this change!!!

**Review by Aaronius — APPROVED** [review]

**Comment by Aaronius** [comment]
Looks great Jon. I'd still like to see a comment on the `assign` in case that was forgotten (https://github.com/adobe/alloy/pull/671#discussion_r583184747). This looks good to go though.

File: src/components/Identity/createComponent.js [hunk 3]
```diff
@@ -30,6 +30,8 @@ export default ({
             setLegacyEcid(ecid);
           }
         }
+        // For sendBeacon requests, getEdge() will return {}, so we are using assign here
+        // so that sendBeacon requests don't override the edge info from before.
         edge = assign(edge, response.getEdge());
```
Comment (Aaronius) [inline src/components/Identity/createComponent.js:35]: I don't care if you change this, but `edge = ` is unnecessary. `assign(edge, response.getEdge());` will suffice.

**Pull request merged**